lib/index.js

```diff
@@ -1,32 +1,11 @@
 'use strict'
-const url = require('url')
 const gitHosts = require('./git-host-info.js')
 const GitHost = module.exports = require('./git-host.js')
 const LRU = require('lru-cache')
+const correctUrl = require('./parse-url.js')
```

Comment:
Is there a compelling reason the function doesn't match the export and file name?

Reply:
No, that was the old name, but I will fixup to match it. Didn't realize I missed that.